ui/pages/notifications/notification-components/index.ts

```diff
@@ -9,9 +9,20 @@ import { components as StakeComponents } from './stake/stake';
 import { components as SwapCompletedComponents } from './swap-completed/swap-completed';
 import { components as LidoWithdrawalRequestedComponents } from './lido-withdrawal-requested/lido-withdrawal-requested';
 import { components as LidoStakeReadyToBeWithdrawnComponents } from './lido-stake-ready-to-be-withdrawn/lido-stake-ready-to-be-withdrawn';
+import { components as SnapNotificationComponents } from './snap/snap';
 
-const { TRIGGER_TYPES } = NotificationServicesController.Constants;
-type TRIGGER_TYPES = NotificationServicesController.Constants.TRIGGER_TYPES;
+export enum SNAP_TRIGGER {
+  SNAP = 'snap',
```

Comment:
Should this be in the notification controller instead? 🤔

Reply:
yeahh...but opening a PR in their repo would take longer. Doing this as a temp workaround.